tests/queries/0_stateless/02891_rename_table_without_keyword.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+db_name=rename_$(date +%s%3N)
+
+$CLICKHOUSE_CLIENT --param_db="$db_name" --multiquery \
+    --query="CREATE DATABASE {db:Identifier};
+             CREATE TABLE IF NOT EXISTS {db:Identifier}.r1 (name String) Engine=Memory();
+             SHOW TABLES FROM {db:Identifier}"
```

Comment:
I think what @tavplubix was referring to (thanks for catching this!) is to use `{CLICKHOUSE_DATABASE:Identifier}` as a param for database name. It is appended by `clickhouse-test` (see [here](https://github.com/ClickHouse/ClickHouse/blob/21e0cb0b18b3be8492f2da071f2b96b9abbed387/tests/clickhouse-test#L1205-L1212)), so you don't need to manually generate anything.

Conveniently we also append `CLICKHOUSE_DATABASE_1` identifier, so you can use that to rename the database to, so the last query can be written like 
```
RENAME {CLICKHOUSE_DATABASE:Identifier} TO {CLICKHOUSE_DATABASE_1:Identifier}; -- { serverError UNKNOWN_TABLE }
```

As a result as we don't need anything generated, we can revert back the tests to the simpler sql file instead of running a shell script in my opinion.

Reply:
Yes, it doesn't make sense for .sh tests because in .sh tests we have `$CLICKHOUSE_DATABASE` env var

Reply:
Ok, I undertand what you mean and I've update the test script.

Reply:
I've tried use `{CLICKHOUSE_DATABASE:Identifier}`, but there were some strange problems. So I think using the current test script should also be ok.

Reply:
I pushed the test using SQL files. Usually we prefer to use SQL files. The reasons I am aware of:
1. They are simpler than the script based tests
2. The queries in SQL scripts are picked up by the fuzzer and used to fuzz ClickHouse.

I hope you don't mind that I pushed a commit, but I think this make the tests better and it was easier for me. On the other hand, if you have any questions regarding to the test file, I am more than happy to answer it. If you want to understand why it wasn't working for you, I am also happy to help you, just show the errors you faced.

Reply:
thx for your helping.